app/forms/nsm/steps/work_item_form.rb

```diff
@@ -8,17 +8,19 @@ class WorkItemForm < ::Steps::BaseFormObject
       attribute :time_spent, :time_period
       attribute :completed_on, :multiparam_date
       attribute :fee_earner, :string
-      attribute :uplift, :integer
+      attribute :uplift, :fully_validatable_integer
 
       validates :work_type, presence: true, inclusion: { in: WorkTypes.values }
       validate :work_type_allowed
       validates :time_spent, presence: true, time_period: true
       validates :completed_on, presence: true,
               multiparam_date: { allow_past: true, allow_future: false }
       validates :fee_earner, presence: true
-      validates :uplift, presence: true,
-              numericality: { only_integer: true, greater_than_or_equal_to: 1, less_than_or_equal_to: 100 },
-              if: :apply_uplift
+      validates :uplift,
+                presence: true,
+                numericality: { allow_blank: true, only_integer: true, greater_than_or_equal_to: 1, less_than_or_equal_to: 100 },
```

Comment:
Doesn't `allow_blank` contradict `presence: true`?

Reply:
So allow_blank in the `numericality` section mean that the numericality errors won't trigger if the item is blank, meaning we can guarantee that if the item is blank it will be the `presence` validation that triggers, meaning that the error will be `blank` rather than `less_than_or_equal_to` etc. It doesn't really matter here, as the error message is currently the same for either type of error, but it can be fiddly when it's not clear which of several different validation rules would apply if the field was blank.

Reply:
Right okay, maybe add a comment for that then?

Reply:
Cool will do